pkg/controller/hosted/controller.go

```diff
@@ -470,13 +473,23 @@ func createHostingManifestWork(managedClusterName string,
 		if err != nil {
 			panic(err)
 		}
+
+		unstructuredObj := &unstructured.Unstructured{}
+		if err := json.Unmarshal(jsonData, unstructuredObj); err != nil {
+			klog.Errorf("failed to unmarshal object %s: %v", string(jsonData), err)
+			continue
+		}
+		// the namespace has been created, should not be managed by manifestWork.
```

Comment:
please also add some details why it should not be managed by manifestwork.
do we need to check the name of the namespace?

Reply:
added.
I don't think need to check the name of namespace here since this controller only handles the hosted mode.